.vscode/open-source/launch.json

```diff
@@ -23,9 +23,8 @@
       "name": "Start RStudio Server",
       "type": "node",
       "request": "launch",
-      "runtimeExecutable": "npx",
+      "runtimeExecutable": "bash",
```

Comment:
I was previously avoiding the use of "bash" to work around a VSCode bug, but if it's working for you now they must've released a fix in a patch already.